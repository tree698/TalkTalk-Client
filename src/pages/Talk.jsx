import React from 'react';
import Footer from '../components/Footer';
import Navbar from '../components/Navbar';
import SelectedDrawing from '../components/SelectedDrawing';
import SelectedTweet from '../components/SelectedTweet';

export default function Talk() {
  return (
    <section className="w-full h-screen flex flex-col">
      <Navbar />
      <div className="mt-6 md:mt-8 lg:mt-10 mb-2 md:mb-4 w-full max-w-[1400px] flex flex-col md:flex-row items-center mx-auto flex-1">
        <SelectedDrawing />
        <SelectedTweet />
      </div>
      <Footer />
    </section>
  );
}
