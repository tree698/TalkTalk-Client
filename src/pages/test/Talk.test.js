import renderer from 'react-test-renderer';
import Talk from '../Talk';

jest.mock('../../components/Navbar');
jest.mock('../../components/SelectedDrawing');
jest.mock('../../components/SelectedTweet');
jest.mock('../../components/Footer');

describe('Home', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Talk />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
