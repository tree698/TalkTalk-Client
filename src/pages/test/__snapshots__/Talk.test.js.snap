// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<section
  className="w-full h-full flex flex-col"
>
  <div
    className="w-full fixed top-0 left-0 bg-white"
  />
  <div
    className="flex flex-col lg:flex-row flex-1 w-full lg:max-w-5xl my-6 md:my-8 lg:my-10 mx-auto pt-[66px]"
  />
</section>
`;
